config/filter.d/apache-common.conf

```diff
@@ -5,10 +5,16 @@
 
 # Load customizations if any available
 after = apache-common.local
+# Uncomment the following line if you use ErrorLog = syslog
+before = common.conf
```

Comment:
You don't get it...

Local configs do not belong to the release (It makes no sense, because will overwrite user config). 
The user should get an advice to create `apache-common.local`.